src/Compilers/CSharp/Test/Semantic/Semantics/RefFieldTests.cs

```diff
@@ -18188,8 +18146,11 @@ struct B
         r.F = ref this; // 2
     }
 }";
-            var comp = CreateCompilation(new[] { source, UnscopedRefAttributeDefinition }, targetFramework: TargetFramework.Net70);
+            var comp = CreateCompilation(new[] { source, UnscopedRefAttributeDefinition });
             comp.VerifyDiagnostics(
+                // (4,18): error CS9064: Target runtime doesn't support ref fields.
+                //     public ref T F;
+                Diagnostic(ErrorCode.ERR_RuntimeDoesNotSupportRefFields, "F").WithLocation(4, 18),
```

Comment:
>Diagnostic(ErrorCode.ERR_RuntimeDoesNotSupportRefFields, "F").WithLocation(4, 18), [](http://example.com/codeflow?start=16&length=82)

If the goal is to verify presence of this diagnostics, then we probably shouldn't be targeting default framework. Consider targeting Net50, for example.

Reply:
On second look, the original test didn't seem to be about the attribute definition being provided in source. I've removed the extra validation I'd added.